builtin/v8/util/adt/set.go

```diff
@@ -0,0 +1,71 @@
+package adt
+
+import (
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/ipfs/go-cid"
+)
+
+// Set interprets a Map as a set, storing keys (with empty values) in a HAMT.
```

Comment:
nit: Since this is code and not just structs it would be nice to see a link to where this is copied from (if it is copied)

Reply:
Yessir!